### DISCUSSION TITLE
Custom Scripts Failing (3.7.0)

### DISCUSSION THREAD

**Discussion by wz4** [netbox-community/netbox#14713] [Help Wanted!]
I cannot get custom scripts to work. I have a very simple example that is failing with the error below.
```

from extras.scripts import *
  
class CreateSwitch(Script):
  
      class Meta:
          name = "Create Switch"
          description = "Create Switch"
          scheduling_enabled = False
  
      device_name = StringVar(
          label="Device Name",
          required=True,
      )
      address = IPAddressVar(
          label="IP Address",
          required=True,
      )
  
  
      def run(self, data, commit):
          device = data['device_name']
          print(device)
```

ERROR

> Type:
> FAILED
> Created at:
> 2024-01-06, 16:41:35
> Exception:
> 
> Traceback (most recent call last):
>   File "/opt/netbox/venv/lib/python3.10/site-packages/django/apps/config.py", line 235, in get_model
>     return self.models[model_name.lower()]
> KeyError: 'contenttype'
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/opt/netbox/venv/lib/python3.10/site-packages/rq/job.py", line 486, in _deserialize_data
>     self._func_name, self._instance, self._args, self._kwargs = self.serializer.loads(self.data)
>   File "/opt/netbox/venv/lib/python3.10/site-packages/django/db/models/base.py", line 2524, in model_unpickle
>     model = apps.get_model(*model_id)
>   File "/opt/netbox/venv/lib/python3.10/site-packages/django/apps/registry.py", line 213, in get_model
>     return app_config.get_model(model_name, require_ready=require_ready)
>   File "/opt/netbox/venv/lib/python3.10/site-packages/django/apps/config.py", line 237, in get_model
>     raise LookupError(
> LookupError: App 'core' doesn't have a 'ContentType' model.
> 
> The above exception was the direct cause of the following exception:
> 
> Traceback (most recent call last):
>   File "/opt/netbox/venv/lib/python3.10/site-packages/rq/worker.py", line 1422, in perform_job
>     self.prepare_job_execution(job, remove_from_intermediate_queue)
>   File "/opt/netbox/venv/lib/python3.10/site-packages/rq/worker.py", line 1351, in prepare_job_execution
>     self.procline(msg.format(job.func_name, job.origin, time.time()))
>   File "/opt/netbox/venv/lib/python3.10/site-packages/rq/job.py", line 520, in func_name
>     self._deserialize_data()
>   File "/opt/netbox/venv/lib/python3.10/site-packages/rq/job.py", line 488, in _deserialize_data
>     raise DeserializationError() from e
> rq.exceptions.DeserializationError

**Comment by markkuleinio** [comment]
Works here (but doesn't output anything as it should return the output, not print it).

How are you trying to run it?

**Reply by wz4** [reply]
Thank you for the response. Running in a dev environment. I got it to work after restarting network-rq service.
